rag/advanced_rag/harness/tools/search.py

```
@@ -277,6 +280,288 @@ async def bm25_search(tools, query: str, kb_ids: list[str] | None = None, top_n:
+            fields,
+            [],
+            {"id": list(chunk_ids)},
+            [],
+            OrderByExpr(),
+            0,
+            len(chunk_ids),
+            search.index_name(tenant_id),
+            [kb_id],
+        )
+        return settings.docStoreConn.get_fields(res, fields) or []
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)